### PR TITLE
test(): Protractor e2e testing

### PR DESCRIPTION
**Pull request by Izak88**

### CODE REVIEW

**Comment by Izak88** [comment]
@jkuri can you give me permissions to set labels. Tnx

File: package.json [hunk 1]
```diff
@@ -116,6 +130,11 @@
     "webpack-dev-server": "^2.5.1",
     "webpack-dll-bundles-plugin": "^1.0.0-beta.5",
     "webpack-merge": "^4.1.0",
-    "zone.js": "0.8.14"
+    "tree-kill": "^1.1.0",
```
Comment (jkuri) [inline package.json]: `tree-kill` and `temp` not needed I think.

File: package.json [hunk 2]
```diff
@@ -116,6 +130,11 @@
...
+    "temp": "^0.8.3",
+    "ts-node": "^3.2.0",
+    "jasmine-spec-reporter": "^4.1.1",
+    "jasmine": "^2.6.0",
+    "zone.js": "0.8.12"
```
Comment (jkuri) [inline package.json]: why downgrade `zone.js`?

File: package.json (hunk 1 above)
Comment (jkuri) [inline package.json]: sorry. didn't saw how you start `protractor`. this is okay.

**Pull request merged**